scripts/install_runtime_dependencies.sh

```diff
@@ -1,10 +1,11 @@
-#!/bin/bash
+#!/bin/sh
 set -euo pipefail
 
-apt-get update
-apt-get install -y curl libdlib19.2 ffmpeg exiftool libheif1 imagemagick
+apk update
+
+apk add curl libheif lapack cblas ffmpeg exiftool imagemagick imagemagick-heic imagemagick-jpeg imagemagick-raw imagemagick-tiff
```

Comment:
As ImageMagick in Alpine repo separates all its libs and plugins, please review the full list of available add-ons and install all image formats and additional libs, which might be useful for us. So far, I found:
- imagemagick-webp
- imagemagick-svg
- imagemagick-jxl
- imagemagick-libs - not sure what are these libs, but maybe they are useful (converters, codecs,...) - please investigate

Reply:
Also, a similar case might be valid for `ffmpeg`: https://pkgs.alpinelinux.org/packages?name=ffmpeg*&branch=edge&repo=&arch=x86_64&maintainer=
However, as we plan to switch to the Jellyfin's ffmpeg, it might be not so critical, but still there might be the case, Jellyfin's ffmpeg is able to use and rely on those add-ons...

Reply:
Done.

I don't think we are able to use `jellyfin-ffmpeg` with `alpine`, because `jellyfin-ffmpeg` binaries are built with `glibc`. `ffmpeg` in `alpine` is compiled with `qsv` and `vaapi` codecs, which are hardware acceleration by Intel and (maybe) AMD CPU. We need to find a way to compile a `ffmpeg` with `cuda` ourselves if we want to support Nvidia.

Reply:
You've mentioned that you can try to build Jellyfin ffmpeg for Alpine from sources. Isn't that an option anymore?
Did you try to search whether someone already ported Jellyfin ffmpeg to Alpine (or tried at least)?

Yes, with QSV (Intel) and VAAPI (AMD), it is a good start, but it would be much better to have more HW and interfaces/protocols supported

Reply:
I don't have any options yet. Will research solutions first after this PR.

Reply:
As a quick search, without [Nvidia supporting alpine](https://github.com/NVIDIA/nvidia-container-toolkit/issues/270) , we can't support Nvidia HW.

Reply:
Sad news(
However, even without NVidia support, Jellyfin still provides a lot more HW support protocols: https://jellyfin.org/docs/general/administration/hardware-acceleration/

I mean, let's merge now with the default ffmpeg + plugins and then it looks to make sense to try replacing it with the Jellyfin's version